pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -892,6 +910,21 @@ func TestPriorityQueue_UpdateNominatedPodForNode(t *testing.T) {
 		t.Errorf("Unexpected diff after updating pods (-want, +got):\n%s", diff)
 	}
 
+	// Attempt to nominate a pod that was deleted from the informer cache.
+	// Nothing should change.
+	q.AddNominatedPod(deletedPodInfo, &framework.NominatingInfo{NominatingMode: framework.ModeOverride, NominatedNodeName: "node1"})
```

Comment:
It's more like a "nonExistingPod".

If we do want to highlight it's a deleted pod, I'd suggest including it initially in the informer cache and then deleting it _explicitly_.

Reply:
renamed